# Holds Metadata and Command for a Motor

# Name
string<=32 name

# Motor Command Data
float64 position_setpoint
float64 velocity_setpoint
float64 feedforward_torque
float64 kp
float64 kd

# Motor Status
bool enable
